batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,192 @@
+            /*
+             * for (String s : pathParts) { System.out.println("parts: " + s); }
+             */
+            if (pathParts[1].equals("create"))
+                create(request, response);
+
+            else if (pathParts[2].equals("edit")) {
+                int id = Integer.parseInt(pathParts[1]);
+                edit(request, response, id);
+            }
+
+        }
+
+        else {
+            fetchData(request, response);
+        }
+
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        // TODO Auto-generated method stub
+        // doGet(request, response);
+        String path = request.getPathInfo();
+        if (path != null) {
+            String[] pathParts = path.split("/");
+
+            if (pathParts[1].equals("createProcess")) {
+                createProcess(request, response);
+
+            }
+
+            else if (pathParts[2].equals("editProcess")) {
+                int id = Integer.parseInt(pathParts[1]);
+                editProcess(request, response, id);
+
+            }
+
+            else if (pathParts[2].equals("deleteProcess")) {
+                int id = Integer.parseInt(pathParts[1]);
+                deleteProcess(request, response, id);
+            }
+
+        } else {
+            fetchData(request, response);
+        }
+
+    }
+
+    private void deleteProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            employeeService.deleteById(id);
+
+            response.sendRedirect("/batch7crud-roll5/employees");
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    private void edit(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            emp = employeeService.fetchById(id);
+            request.setAttribute("employee", emp);
+            RequestDispatcher dispatcher = request.getRequestDispatcher("/WEB-INF/views/editEmployee.jsp");
+            dispatcher.forward(request, response);
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+
+    private void editProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        // TODO Auto-generated method stub
+        try {
+            emp.setFirstName(request.getParameter("firstName"));
+            emp.setLastName(request.getParameter("lastName"));
+            emp.setDepartment(request.getParameter("department"));
+            emp.setAddress(request.getParameter("address"));
+
+            employeeService.edit(emp, id);
+
+            response.sendRedirect("/batch7crud-roll5/employees");
+
+        } catch (Exception e) {
+            // TODO Auto-generated catch block
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)